src/qe_tools/_constants.py

```diff
@@ -23,6 +23,9 @@
     timeau_to_sec=2.418884326155573e-17,
     invcm_to_THz=0.0299792458,
 
+    ## Values taken from https://gitlab.com/QEF/q-e/-/blob/develop/Modules/constants.f90?ref_type=heads#L66
+    ha_SI=4.3597447222071e-18, # J
```

Comment:
Our convention is to keep [snake case](https://en.wikipedia.org/wiki/Snake_case) for variable names:

```suggestion
    ha_si=4.3597447222071e-18, # J
```